packages/error-reporting/src/interfaces/manual.js

```diff
@@ -74,6 +74,10 @@ function handlerSetup(client, config) {
     }
 
     if (err instanceof ErrorMessage) {
+      var fauxError = new Error(err.message);
```

Comment:
After discussion offline it sounds like there may be an efficient way to get creation site stack traces? This behavior will mirror that of native JS errors.